terra-jupyter-base/scripts/kernel/kernel_bootstrap.sh

```diff
@@ -13,6 +13,6 @@
 
 # The workspace name is simply the CWD of the running notebook.
 PWD="$(pwd)"
-export WORKSPACE_NAME=`basename $(dirname $(pwd))`
```

Comment:
I think the second quote is off...it should be after `)`

Reply:
Right you are, fixed